src/plone/restapi/tests/test_blocks_serializer.py

```diff
@@ -189,3 +192,95 @@ def test_serialize_subblocks_transformers(self):
+                "value": [
+                    {
+                        "children": [
+                            {"text": "this is a "},
+                            {
+                                "children": [
+                                    {"text": ""},
+                                    {
+                                        "children": [{"text": "slate link"}],
+                                        "data": {
+                                            "link": {
+                                                "internal": {
+                                                    "internal_link": [resolve_uid_link]
+                                                }
+                                            }
+                                        },
+                                        "type": "a",
+                                    },
+                                    {"text": ""},
+                                ],
+                                "type": "strong",
+                            },
+                            {"text": " inside some text"},
+                        ],
+                        "type": "p",
+                    }
+                ],
+            },
+            "6b2be2e6-9857-4bcc-a21a-29c0449e1c68": {"@type": "title"},
+        }
+
+        res = self.serialize(
+            context=self.portal["doc1"],
+            blocks=blocks,
+        )
+
+        value = res["2caef9e6-93ff-4edf-896f-8c16654a9923"]["value"]
+        link = value[0]["children"][1]["children"][1]
+        resolve_link = link["data"]["link"]["internal"]["internal_link"][0]["@id"]
+
+        self.assertTrue(resolve_link == self.portal.absolute_url() + "/doc1")
+
+    def test_simple_link_serializer(self):
+        doc_uid = IUUID(self.portal["doc1"])
+        resolve_uid_link = "../resolveuid/{}".format(doc_uid)
+
+        blocks = {
+            "abc": {
+                "@type": "slate",
+                "plaintext": "Frontpage content here",
+                "value": [
+                    {
+                        "children": [
+                            {"text": "Frontpage "},
+                            {
+                                "children": [{"text": "content "}],
+                                "data": {
+                                    "url": resolve_uid_link,
+                                },
+                                "type": "link",
+                            },
+                            {"text": "here"},
+                        ],
+                        "type": "h2",
+                    }
+                ],
+            }
+        }
+        res = self.serialize(
+            context=self.portal["doc1"],
+            blocks=blocks,
+        )
+        value = res["abc"]["value"]
+        link = value[0]["children"][1]["data"]["url"]
+        self.assertTrue(link, self.portal.absolute_url() + "/doc1")
+
+    def test_bogus(self):
```

Comment:
@tiberiuichim huh? What's that? :)

Reply:
hmm... not sure how that ended up there. I thought someone else added it :)